modules/custom/activity_creator/src/ActivityNotifications.php

```diff
@@ -116,10 +120,12 @@ public function markEntityAsRead(AccountInterface $account, EntityBase $entity)
    * @param \Drupal\activity_creator\Entity\Activity $activity
    *   Activity object.
    * @param int $status
-   *   See: activity_creator_field_activity_status_allowed_values().
+   *   See: activity_creator_field_activity_status_allowed_values()
    *
-   * @return \Drupal\activity_creator\Entity\Activity
+   * @return int
```

Comment:
Is this correct? Even if it's not the Activity object I would expect a `bool` instead of `int`.

Reply:
Updated.